src/generators/generator.py

```diff
@@ -380,7 +380,7 @@ def gen_class_decl(self,
         is_final = ut.random.bool() and class_type == \
             ast.ClassDeclaration.REGULAR
         type_params = type_params or self.gen_type_params(
-            with_variance=self.language == 'kotlin')
+            with_variance=self.language in ['kotlin', 'scala'])
```

Comment:
I think this should be handled by the config, but it can be refactored later. Not related to this PR.